Example/Example/ComicsUI.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  ComicsUI.swift
+//  Example
+//
+//  Created by Davide Mendolia on 19/11/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import BothamUI
+
+protocol ComicsUI: BothamUI {
+    func showItems(items: [Comic])
```

Comment:
Ok here is tricky. func showItems(items: [Comic]) have a default implementation in ColectionViewController.
I prefer showComics() also but if a I do that. I need to add the following in ComicsViewController. 
```swift
    func showComics(comics: [Comic]) {
        self.showItems(comics)
    }
```
It's a little bit more verbose, less magic, but maybe better.